File: app/src/main/java/com/sagar/watchnext/network/models/people/tvcredits/TvCredits.java

package com.sagar.watchnext.network.models.people.tvcredits;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class TvCredits {

    @SerializedName("cast")
    private List<Cast> cast;

    @SerializedName("crew")
    private List<Crew> crew;

    @SerializedName("id")
    private int id;

    public List<Cast> getCast() {
        return cast;
    }

    public List<Crew> getCrew() {
        return crew;
    }

    public int getId() {
        return id;
    }
}

File: app/src/main/java/com/sagar/watchnext/network/models/people/tvcredits/TvCredits.kt
package com.sagar.watchnext.network.models.people.tvcredits

import com.google.gson.annotations.SerializedName

class TvCredits {

    @SerializedName("cast")
    val cast: List<Cast>? = null

    @SerializedName("crew")
    val crew: List<Crew>? = null

    @SerializedName("id")
    val id: Int = 0
}
